test/rules/unused_macros/bad.exs

```diff
@@ -0,0 +1,2 @@
+defmodule MeandroTest.MyMacroTest do
```

Comment:
This doesn't look like a `bad` module to me. It seems more like a `none.exs`.

Reply:
That one is actually not used. Deleted